library/Icinga/Web/Navigation/Renderer/RecursiveMenuNavigationRenderer.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/* Icinga Web 2 | (c) 2025 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Web\Navigation\Renderer;
+
+use Icinga\Web\Navigation\NavigationItem;
+
+/**
+ * Renderer for the multi level navigation in the sidebar menu
+ */
+class RecursiveMenuNavigationRenderer extends RecursiveNavigationRenderer implements NavigationRendererInterface
+{
+    public function beginChildren(): void
+    {
+        parent::beginChildren();
+
+        $item = new NavigationItem($this->getInnerIterator()->current()->getParent()->getName());
+        $item->setCssClass('nav-item-header');
+        $item->render();
```

Comment:
1. You're still ignoring the label.
2. The render result is ignored. You're rendering twice at the moment.

Reply:
Remember to hide the label's badge using CSS.